src/video_provider_ffmpegsource.cpp

```diff
@@ -248,6 +248,9 @@ void FFmpegSourceVideoProvider::LoadVideo(agi::fs::path const& filename, std::st
 			throw VideoOpenError(std::string("Failed to set input format: ") + ErrInfo.Buffer);
 		ColorSpace = colormatrix_description(FFMS_CS_BT470BG, CR);
 	}
+	else {
```

Comment:
I think with curly braces the following would be according to the style:
```c++
} else {
    // code
}
```
Also pass on `&ErrInfo` a la the other call and throw in a similar vein in case of failure. I would otherwise like this more if this would only happen in case `CS == FFMS_CS_UNSPECIFIED`, as that's what this is **really** trying to fix. That said, that would require actual re-thinking of this logic so I'm not going to go that nazi :P .